src/components/scenes/RequestScene.tsx

```diff
@@ -439,20 +439,17 @@ export class RequestSceneComponent extends React.Component<Props, State> {
     const subject = wallet != null ? sprintf(lstrings.request_email_subject, config.appName, wallet.currencyInfo.displayName) : ''
     const message = `${sharedAddress}${addOnMessage}`
 
-    const shareOptions = {
+    const shareOptions: ShareOptions = {
       subject,
       message: Platform.OS === 'ios' ? message : message + edgePayUri,
-      url: Platform.OS === 'ios' ? edgePayUri : ''
+      url: Platform.OS === 'ios' ? edgePayUri : '',
+      failOnCancel: false
     }
 
-    Share.open(shareOptions).catch(e => console.log(e))
+    await Share.open(shareOptions).catch((e: unknown) => showError(e))
```

Comment:
You can do `.catch(showError)` directly, since the function is designed to be used that way.